extensions/large-type/package.json

```diff
@@ -7,21 +7,65 @@
   "icon": "command-icon.png",
   "author": "gbarba",
   "contributors": [
-    "FezVrasta"
+    "FezVrasta",
+    "garrill"
   ],
   "license": "MIT",
   "commands": [
     {
-      "name": "index",
-      "title": "Show Large Type",
-      "description": "Large Type allows you to show text in large characters across your screen",
+      "name": "enter-text",
```

Comment:
Can we keep the old `name`? It is the unique ID of the command where ranking, aliases, and hotkeys are saved. You can, of course, change the title 🙂

Reply:
Sure, I din't realise this is how it worked. Will revert the name